spirv_msl.cpp

```diff
@@ -9322,6 +9473,12 @@ void CompilerMSL::emit_instruction(const Instruction &instruction)
 
 		break;
 	}
+	case OpSetMeshOutputsEXT:
+	{
+		flush_variable_declaration(builtin_primitive_indices_id);
+		statement("spvMesh.set_primitive_count(", to_unpacked_expression(ops[1]), ");");
```

Comment:
How does this handle rules where the first invocation of the workgroup wins?

Reply:
It doesn't in this draft. Metal spec is unclear about access from multiple threads.
Also emulating vulkan spec here is, unfortunately, another tough workaround: would require something like `subgroupElect`, but for entire workgroup.

Reply:
> subgroupElect

I didn't realize, back than, that Vulkan requires uniform control flow. Fixed now.